apache/docker-entrypoint.sh

```diff
@@ -72,6 +72,25 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
   : "${ROUNDCUBEMAIL_TEMP_DIR:=/tmp/roundcube-temp}"
   : "${ROUNDCUBEMAIL_REQUEST_PATH:=/}"
 
+  if [ ! -z "${ROUNDCUBEMAIL_INSTALL_PLUGINS}" ]; then
+    echo "Installing plugins from the list"
+    echo "Plugins: ${ROUNDCUBEMAIL_INSTALL_PLUGINS}"
```

Comment:
Wouldn't it be better to print `${ROUNDCUBEMAIL_PLUGINS}` here?

Reply:
Indeed, I changed this